source/Grid/Grid.js

```diff
@@ -472,6 +479,7 @@ export default class Grid extends Component {
         className={cn('Grid', className)}
         onKeyDown={this._onKeyPress}
         onScroll={this._onScroll}
+        onWheel={()=>{}}
```

Comment:
So FF and Chrome seem to exhibit different behavior if an onWheel handler is registered, specifically, without it the new scroll position repaints, _before_ the `onScroll` event fires. Presumably the browser is doing some sort of off-thread optimization for wheel scrolls that it has to bail out of when you listen for the wheel event.

That is just me pulling guesses out of the air though, I couldn't find anything explaining why it works. I do see a noticeable lag in the scroll updates of the other grids though without it.